spec/controllers/users_controller_spec.rb

```diff
@@ -70,4 +108,29 @@
       expect(assigns(:user)).to eq(user)
     end
   end
-end
\ No newline at end of file
+
+  describe "GET new_with_invitation_token" do
+    it "renders the :new view template" do
+      invitation = Fabricate(:invitation)
+      get :new_with_invitation_token, token: invitation.token
+      expect(response).to render_template :new
+    end
+
+    it "sets the recipients email address" do
+      invitation = Fabricate(:invitation)
+      get :new_with_invitation_token, token: invitation.token
+      expect(assigns(:invitation_token)).to eq(invitation.token)
+    end
+
+    it "sets @invitation_token" do
```

Comment:
Neither does this one, it looks like a copy and pasted version of the spec 2 above.